doc/cli.md

```diff
@@ -126,10 +126,10 @@ Option | Description
 `--input`, `-i` (required) | Path to input profile.
 `--map`, `-m` (required) | Path to mapping of source features to target features.
 `--output`, `-o` (required) | Path to output profile.
-`--frac`, `-f` | Count each target feature as 1 / _k_ (_k_ is the number of targets mapped to a source). Otherwise, count as one.
+`--divide`, `-d` | Count each target feature as 1 / _k_ (_k_ is the number of targets mapped to a source). Otherwise, count as one.
+`--field`, `-f` | Index of field to be collapsed in a stratified profile. For example, use `-f 2` to collapse "gene" in "microbe\|gene".
```

Comment:
Perhaps a future feature update where the field can be a name like 'gene' instead of an index. 

Index is fine for now, but this is something to consider for ease of use for end users.

Reply:
Good point! One reason this hasn't been implemented is that QIIME 2 has mandatory index headers (like `#FeatureID`). We need to think about where else we can store these field definitions.